app/controllers/api/v1/graphql_controller.rb

```diff
@@ -53,7 +53,7 @@ def parse_graphql_result
         # Mutations are not batched, so we can return errors in the root
         rescue ActiveRecord::RecordInvalid, RuntimeError, NameError, GraphQL::Batch::NestedError => e
           @output = parse_json_exception(e)
-          CheckSentry.notify(e)
+          CheckSentry.notify(e) unless ActiveRecord::RecordInvalid
```

Comment:
@vasconsaurus you need to check that `e` is an exception of that type. When you just check for the constant itself, it will always return `true` since it's defined:

```
> !!ActiveRecord::RecordInvalid
=> true
```

There are different ways to check for the class of an object, for example using `is_a?` or `kind_of?`:

```
> e = StandardError.new
=> #<StandardError: StandardError>

> e.is_a?(ActiveRecord::RecordInvalid)
=> false

> e.kind_of?(ActiveRecord::RecordInvalid)
=> false

> e = ActiveRecord::RecordInvalid.new
=> #<ActiveRecord::RecordInvalid: Record invalid>

> e.is_a?(ActiveRecord::RecordInvalid)
=> true

> e.kind_of?(ActiveRecord::RecordInvalid)
=> true
```

Reply:
derp! of course! will update it